clouddriver-event/src/main/kotlin/com/netflix/spinnaker/clouddriver/event/EventMetadata.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.clouddriver.event
+
+import java.time.Instant
+
+/**
+ * Metadata for a [SpinEvent].
+ *
+ * @param sequence Auto-incrementing number for event ordering
+ * @param originatingVersion The aggregate version that originated this event
+ * @param timestamp The time at which the event was created
+ * @param serviceVersion The version of the service (clouddriver) that created the event
+ * @param provenance Where the event was generated and by what
+ */
+data class EventMetadata(
+  val sequence: Long,
+  val originatingVersion: Long,
+  val timestamp: Instant = Instant.now(),
+  val serviceVersion: String = "unknown",
+  val provenance: String = "unknown"
```

Comment:
this might be just me, but i hate the word `provenance`. it always takes me a few seconds to think about what it means. I am going to go on a limb and say that, esp for developers whose primary language is not English, will find this work similarly difficult to parse.
I think `source` or `origin` convey the same information and are much more widely used